scripts/firedrake-install

```diff
@@ -1250,7 +1255,7 @@ for opt in ["mpicc", "mpicxx", "mpif90"]:
     os.symlink(src, dest)
 
 if options.get("mpiexec"):
-    mpiexec_loc = shutil.which(args.mpiexec)
+    mpiexec_loc = shutil.which(options.get("mpiexec"))
```

Comment:
`firedrake-update` will fail here if you previously installed with manually specified system MPI, since `args.mpiexec` is not present, but `options['mpiexec']` is